src/site/_filters/find-by-url.js

```diff
@@ -68,12 +68,7 @@ const findByUrl = (url) => {
     throw new Error(`No collection has been memoized yet.`);
   }
 
-  const found = memo[url];
-  if (!found) {
```

Comment:
Path definitions are now common to all languages. If a given post is not translated yet, the file will be missing in that language, which is fine, so I think we can be less strict about errors in case of missing files.

Reply:
ok. I'm curious if this might lead to other files requesting a post by url, getting back null, and trying to act on it. But maybe we can see if that happens and address it then.